cli/provider/cmd.go

```diff
@@ -387,6 +388,14 @@ func (c *CmdConfigurator) ValidateFlags(ctx context.Context, cmd *cobra.Command)
 
 		c.cfg.Path = absPath + "/keploy"
 		if cmd.Name() == "test" {
+			//check if the keploy folder exists
+			if _, err := os.Stat(c.cfg.Path); os.IsNotExist(err) {
+				recordCmd := models.HighlightGrayString("keploy record")
+				errMsg := fmt.Sprintf("Keploy folder not found. Please record testcases using %s command", recordCmd)
+				utils.LogError(c.logger, err, errMsg)
```

Comment:
Hey just saw the screenshot and I think the error shouldn't include stat error and it will be good if error is "no test-sets found" rather than "no keploy folder" because keploy folder has no usecase to user but test-sets does. Please update the screenshots after this change.

Reply:
Refactored and updated screenshot no.3